js/dashboard.js

```diff
@@ -940,7 +940,7 @@ var Dashboard = {
 
       return $.ajax({
          url:CFG_GLPI.root_doc+"/ajax/dashboard.php",
-         method: 'POST',
+         method: 'GET',
```

Comment:
The URI length limit was the reason it was switched to POST. Since the PR was focused on performance only, I didn't try reworking the code to send less data in the request.